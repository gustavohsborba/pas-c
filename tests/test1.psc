begin
end
_isNotAndIdentifier
ThisIs1id