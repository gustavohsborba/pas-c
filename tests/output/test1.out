Opening file tests/test1.psc...
ERROR in line 4, column 6: Sintax error found - Expected statement, but '' found
